grammars/c#.cson

```diff
@@ -263,6 +263,10 @@
         'name': 'keyword.other.source.cs'
       }
       {
+        'match': '\\b(var)\\b'
+        'name': 'storage.type.var.cs'
```

Comment:
Since all the other rules have `.source` attached, I think it would be best to also attach that here and then remove all of them in a separate PR for consistency.